dist/alpine/Dockerfile

```
@@ -1,34 +1,37 @@
 ARG BASE_IMAGE_HASH
 ARG PY_VER
+FROM datajoint/djbase:py${PY_VER}-alpine-${BASE_IMAGE_HASH} as test_build
 
-FROM datajoint/djbase:py${PY_VER}-alpine-${BASE_IMAGE_HASH} as jup_build
-USER dja:anaconda
-COPY --chown=dja:anaconda ./config/pip_requirements.txt /tmp/
+LABEL maintainerName="Raphael Guzman" \
+	maintainerEmail="raphael@datajoint.com" \
+	maintainerCompany="DataJoint"
```

You can update this to be more generic than calling me out specifically. :sweat_smile: 

Perhaps:

```suggestion
LABEL maintainerName="DataJoint Contributors" \
	maintainerEmail="service-health@datajoint.com" \
	maintainerCompany="DataJoint"
```